package.json

```diff
@@ -10,6 +10,7 @@
     "babel-polyfill": "^6.26.0",
     "blockies": "^0.0.2",
     "blockies-identicon": "^0.1.0",
+    "crypto-js": "^3.1.9-1",
```

Comment:
I think we can use web3 instead of this library. No?

Reply:
or maybe use the library `ethers.js`, see https://docs.ethers.io/ethers.js/html/api-utils.html#cryptographic-functions

Reply:
no it is a different sha3 algorithm. I tried that one first. Pretty confusing that `web3.eth.sha3` is different than the Solidity `keccak256`. I can try the `ethers.js keccak256`. Should work

Reply:
ok you can try `ethers.js`